changelog.md

```diff
@@ -61,26 +97,54 @@ We now publish a separate marathon-plugin-interface.jar with every Marathon rele
 This artifact holds all the inerfaces needed to develop your own Marathon plugin.
 
 #### Consolidate logging to use slf4j
-We moved completely to slf4j as Logging API. The logging backend still uses log4j.
+We moved completely to slf4j as Logging API. 
+
+#### Several performance improvements
+* A separate thread pool is used for health check operations
+* Efficiently create groups/apps 
 
 #### Introduce configurable zk node compression
 We introduced zk compression which improves performance significantly. Compression can be turned on/off via cmd line flags and is enabled by default.
 
+#### Tasks are now stored via the standard EntityRepository
+The storage of tasks was always handled specifically in Marathon.
```

Comment:
Ah, I see. In that case I suggest:
The storage of tasks was handled separately in previous versions of Marathon.